src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar month-length">
+      <div class="calendar_day start-day-tue"></div>
```

Comment:
```suggestion
    <div class="calendar month-length start-day-tue">
      <div class="calendar_day"></div>
```

Використовувати :first-child і задавати для нього марджин. Відкоригуй свій цикл:
```
 .start-day-#{$day} :first-child
```

Reply:
должен ли class start-day меняться? и если да то как?

Reply:
Все равно не понимаю как сделать так что бы модификатор реагировал на первого ребенека и при этом в коде был модификатор start-day